### PR TITLE
Add cypress black box code cov instrumentation to ui_next 

### PR DESCRIPTION
**Pull request by unlikelyzero**
##### SUMMARY
This PR adds the capability to run cypress tests and collected code coverage metrics from an instrumented ui_next dev build. This generated report matches the code cov report generated by jest.

This PR does not include documentation on how to run this locally as that will exist alongside the testing codebase for now.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION

### CODE REVIEW

File: awx/ui_next/package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
   },
   "scripts": {
     "start": "PORT=3001 HTTPS=true DANGEROUSLY_DISABLE_HOST_CHECK=true react-scripts start",
+    "start-instrumented": "DEBUG=instrument-cra PORT=3001 HTTPS=true DANGEROUSLY_DISABLE_HOST_CHECK=true react-scripts -r @cypress/instrument-cra start",
```
Comment (jlmitch5) [inline awx/ui_next/package.json:57]: is this starting the ui server or starting the cypress stuff?  if it's the later, it should be named something else.  if it's just starting a special version of the ui server this name is fine

Comment (jlmitch5) [reply awx/ui_next/package.json:57]: sounds like the former is the case, ignore the comment

**Comment by unlikelyzero** [comment]
@jlmitch5 @jakemcdermott can this be merged? I don't want it hanging around for too long :octocat:

**Comment by unlikelyzero** [comment]
June was a long time ago. I'm going to rebase and retest this.

**Review by mabashian — APPROVED** [review]

**Pull request merged**